listeners.go

```diff
@@ -39,28 +40,31 @@ type krazyServer struct {
 
 var (
 	listenersMu sync.Mutex
-	listeners   = make(map[string][]*krazyServer) // by private IP address or unix socket path
+	listeners   = make(map[string][]*krazyServer) // by unix socket path and private IP address
 )
 
-// gokrazyHTTPUnixSocket is the unix socket path that
-// the HTTP server listens on when httpPassword is empty.
-const gokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
+// GokrazyHTTPUnixSocket is the unix socket path that the HTTP server listens on.
+const GokrazyHTTPUnixSocket = "/run/gokrazy-http.sock"
 
 // tlsConfig: tlsConfig. nil, if the listeners should not use https (e.g. for redirects)
 func updateListeners(port, redirectPort string, tlsEnabled bool, tlsConfig *tls.Config) error {
-	var hosts []string // private IP addresses or unix socket path
-	var err error
-
+	// Always listen on a Unix socket.
+	//
+	// Local packages running as root can connect through the Unix socket
```

Comment:
Change “packages” to “processes” please. You are correct in that gokrazy deals with Go packages, but at the level we are talking about here (runtime), the distinction is executable files that get loaded into operating system processes :)